.github/workflows/style.yaml

```diff
@@ -0,0 +1,39 @@
+
+name: Style R scripts and Rmd notebooks
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  pull_request:
+    branches: [ main ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "style"
+  style:
+    runs-on: ubuntu-latest
+    container:
+      image: rocker/tidyverse:4.2
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+        # What branch to commit to: the one from the pull request
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install packages
+        run: Rscript -e "install.packages('styler')"
+
+      - name: Run styler
+        run: Rscript -e "styler::style_file(list.files(pattern = 'Rmd$|.R$', recursive = TRUE, full.names = TRUE));warnings()"
+      
+      - name: Configure Git repository
+        run: |
+          git config --global --add safe.directory /__w/AutoGVP/AutoGVP
```

Comment:
the action ran, changed two files, but no changes were detected, so I had to add this bit to get it to work. (shown in the error message under commit style changes [here](https://github.com/diskin-lab-chop/AutoGVP/pull/75/checks?sha=9eb692d9ca0a2ffad2107692a8d23808101246dd))

Reply:
That's interesting, cause the checkout action is supposed to be doing that already. I think this will still work, but I wouldn't mind taking a crack at figuring out what's happening here before you merge.

<img width="682" alt="image" src="https://github.com/diskin-lab-chop/AutoGVP/assets/12488263/50dad9c4-a5e6-4d5a-8437-28dc4af05681">